### DISCUSSION TITLE
Question about the process

### DISCUSSION THREAD

**Discussion by smchargue** [pnp/sp-dev-fx-webparts#2448] [Q&A]
Mostly out of curiosity.  Why does the solution require everyone, especially non-developers who just what the basic ootb functionality of the component, to download and go through a development process to generate the sppfx file? 

Is there a particular reason why a sppfx file isn't made available as a downloaded and installable file?  Is there any reason why, once i do compile the package, that I couldn't use it for multiple tenants? 

btw - this questoin is specifically about the react-script-editor web part, if that wasn't clear.

**Comment by franklin050187** [comment]
Hi there, security reason probably.
I would not upload unknown package to SharePoint, when you are building it you can at least check the content.
